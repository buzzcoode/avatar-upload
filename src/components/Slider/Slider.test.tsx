import React from 'react';
import { render } from '@testing-library/react';
import { Slider } from '.';

it('renders correctly', async () => {
  const { queryByTestId } = render(<Slider />);
  expect(queryByTestId('styled-slider')).toBeTruthy();
});

it('renders snapshot', async () => {
  const { container } = render(<Slider />);
  expect(container).toMatchSnapshot();
});