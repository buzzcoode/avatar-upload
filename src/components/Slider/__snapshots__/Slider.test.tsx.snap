// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders snapshot 1`] = `
.emotion-0 {
  width: 100%;
}

<div>
  <input
    class="emotion-0"
    data-testid="styled-slider"
    type="range"
  />
</div>
`;
